src/content/docs/synthetics/synthetic-monitoring/administration/synthetics-api.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Manage synthetic monitors via API
+tags:
+  - Synthetics
+  - Synthetic monitoring
+  - Administration
+metaDescription: 'For New Relic synthetic monitoring, how to use APIs to manage monitors.'
+redirects:
+---
+
+We have two options for managing your New Relic synthetic monitors via API: 
+
+* NerdGraph (recommended): this is the primary and recommended way. See [our NerdGraph synthetics tutorial](/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial).
+* REST API: Some customers prefer to use our older REST API. See [Synthetics REST API](/docs/apis/synthetics-rest-api/monitor-examples/manage-synthetics-monitors-rest-api)
```

Comment:
I would x2 check if it's worth calling this out. If we do want it listed, might want to include a note that we recommend NerdGraph because we might not support REST... but confirm with SME